modules/ossm-kiali-ossmc-plugin-user-guide.adoc

```diff
@@ -0,0 +1,56 @@
+////
+This CONCEPT module included in the following assemblies:
+-service_mesh/v2x/ossm-kiali-ossmc.adoc
+////
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-kiali-ossmc-plugin-user-guide_{context}"]
+= About the OpenShift Service Mesh Console plugin
+//In the title include nouns or noun phrases that are used in the body text.
+//Do not start the title of concept modules with a verb..
+
+
+// Working on converting this to Asciidoc
+// Commenting out so there is delineation in this file 
+// DESCRIPTION LIST in asciidoc
+// -- are an open  block, help keep things together esp with bullets in the middle of stuff
+// plugin compatible with OCP 4.15+
+
+[WARNING]
+The OpenShift Service Mesh Console (OSSM) plugin only supports a single tenant. Whether that tenant is configured to access only a subset of OpenShift projects or has access cluster-wide to all projects does not matter, however, only a single tenant can be accessed.
+
+The {SMProductName} (OSSMC) plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh from the OpenShift web console. You can install the OSSMC plugin in only one of two ways: using the OpenShift web console, or through the CLI.
+
+Installing the OSSMC plugin creates a new category, *Service Mesh*, in the main OpenShift web console navigation. Click *Service Mesh* to see:
+
```

Comment:
Unless you were explicitly asked to document each of the menu items in the UI, I would advise against it. While technically accurate, it is best practice to focus on user tasks in instead of describing the UI. In your PR description, you mention that end user documentation will be provided elsewhere. And in fact, the user guide already seems to cover this. So will that resource also be made available to users now? Or will you be converting that guide and this content is being provided, until additional information can be converted?

If you must include the following information, I suggest removing it from this module and moving it to a "Using" section.  From a user perspective, I have not installed the plugin yet, but I am now presented with information on all of the elements in the UI.

I am not sure how pressed you are for time, but I thought this was definitely worth mentioning.

Reply:
Mike and I had a chat. The UI is documented because this plugin also exists in the community and may differ from what Red Hat supports. The documented UI is what Red Hat supports.